index.js

```diff
@@ -221,15 +222,28 @@ async function checkHeartbeat() {
         const innosentData = await redis.getLatestInnosentSensorData(location.locationid)
         latestRadar = moment(innosentData.timestamp, 'x')
       }
+
+      const doorData = await redis.getLatestDoorSensorData(location.locationid)
+      latestDoor = moment(doorData.timestamp, 'x')
       const currentTime = moment()
```

Comment:
What do you think about getting the current time for this comparison from Redis: https://redis.io/commands/TIME ? This will ensure that we're comparing apples to apples in case our server times get out of sync.

Reply:
I like it, and moment is deprectaed anyways